src/modules/mahjong/utils/functions/checkTypes.ts

```diff
@@ -0,0 +1,16 @@
+import SimpleTileTypes from '../../enums/SimpleTileEnums';
+import BonusTileTypes from '../../enums/BonusTileEnums';
+import HonorTileTypes from '../../enums/HonorTileEnums';
+import TileMapper from '../../Tile/map/TileMapper';
+
+export const isSimpleTile = (key: string): boolean =>
+  // eslint-disable-next-line
+  Object.values(SimpleTileTypes).includes(<SimpleTileTypes>TileMapper[key].type);
+
+export const isHonorTile = (key: string): boolean =>
+  // eslint-disable-next-line
+  Object.values(HonorTileTypes).includes(<HonorTileTypes>TileMapper[key].type);
+
+export const isBonusTile = (key: string): boolean =>
+  // eslint-disable-next-line
+  Object.values(BonusTileTypes).includes(<BonusTileTypes>TileMapper[key].type);
```

Comment:
no curly braces for these functions? didn't know you can do that lol

Reply:
it's technically still considered one line lol